plugin-packs/postcss-preset-env/test/import.ch88.expect.css

```diff
@@ -1,8 +1,9 @@
 .test-custom-properties {
+	order: 1;
 	order: var(--order);
 }
 
-@media (max-width: env(--sm)) {
+@media (max-width: 40rem) {
 	.test-custom-media-queries {
 		order: 2;
 	}
```

Comment:
These are the plugins in question.
They didn't run before for modern chrome, but they do now.